apple-pie/index.html

```diff
@@ -4,18 +4,70 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- don't forget to link your styles -->
+    <link rel="stylesheet" href="/apple-pie/styles/style.css">
   </head>
   <body>
-    Apple Pies This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
+    <div class="background-image">
+<h1 class="title">Apple Pie</h1>
+    </div>
+    <div>
+<section>
```

Comment:
A `<div>` or a `<section>` do the exact same thing, it's just a matter of semantics/good practices, and it is important for people that use screen readers. 
So there is no need to open a `<div>` tag every time you have a `<section>` you can just use the `<section>`:

This:
```
    <div>
      <section>
        <p>
          This was my grandmother's apple pie recipe. I have never seen another
          one <br />
          quite like it. It will always be my favorite and has won me several
          first place <br />
          prizes in local competitions. I hope it becomes one of your favorites
          as well!
        </p>
      </section>
    </div>
```

Is the same as this:
```
      <section>
        <p>
          This was my grandmother's apple pie recipe. I have never seen another
          one <br />
          quite like it. It will always be my favorite and has won me several
          first place <br />
          prizes in local competitions. I hope it becomes one of your favorites
          as well!
        </p>
      </section>
```